lasagne/layers/merge.py

```diff
@@ -11,6 +13,232 @@
 ]
 
 
+def autocrop(inputs, cropping):
+    """
+    Crops the given input arrays.
+
+    Cropping takes a sequence of inputs and crops them per-axis in order to
+    ensure that their sizes are consistent so that they can be combined
+    in an element-wise fashion. If cropping is enabled for a specific axis,
+    the minimum size in that axis of all inputs is computed, and all
+    inputs are cropped to that size.
+
+    The per-axis cropping modes are:
+
+    `None`: this axis is not cropped, inputs are unchanged in this axis
+
+    `'lower'`: inputs are cropped choosing the lower portion in this axis
+    (`a[:crop_size, ...]`)
+
+    `'upper'`: inputs are cropped choosing the upper portion in this axis
+    (`a[-crop_size:, ...]`)
+
+    `'center'`: inputs are cropped choosing the central portion in this axis
+    (`a[offset:offset+crop_size, ...]` where
+    `offset = (a.shape[0]-crop_size)//2)
```

Comment:
You seem to miss the second apostrophe in that line. (And apart from that, it should be double apostrophes for code fragments. Single apostrophes are for referring to argument or variable names.) To be sure, run `make html` in the `docs` directory and open `_build/html/modules/layers/merge.html` in a browser to check whether it renders fine.